apps/morph_stats

```diff
@@ -84,32 +81,38 @@ def get_parser():
                         help=('Summary output file name, if it ends in .json, '
                               'a json file is created, if .csv, then a csv file'))
 
+    parser.add_argument('-I', '--ignored-exceptions', dest='ignored_exceptions', default='',
```

Comment:
How would I know what to provide here?
Maybe, use `choices` here and remove `if hasattr` check on line 104?